src/Schema/Builder.php

```diff
@@ -244,6 +249,26 @@ public function getIndexes($table)
             ];
         }
 
+        try {
+            $indexes = $collection->listSearchIndexes(['typeMap' => ['root' => 'array', 'array' => 'array', 'document' => 'array']]);
+            foreach ($indexes as $index) {
+                $indexList[] = [
+                    'name' => $index['name'],
+                    'columns' => match ($index['type']) {
+                        'search' => array_keys($index['latestDefinition']['mappings']['fields'] ?? []),
+                        'vectorSearch' => array_column($index['latestDefinition']['fields'], 'path'),
```

Comment:
Per [Atlas Vector Search Index Fields](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-type/#atlas-vector-search-index-fields), I expect this might return field paths instead of field names. Is that going to cause problems with whatever consumes this?

I assume it may be consistent with how the basic `listIndexes()` query above runs (using `array_keys($index->getKey())` but I'm not sure.